backend/dataall/cdkproxy/stacks/policies/cloudformation.py

```diff
@@ -3,7 +3,7 @@
 
 
 class Cloudformation(ServicePolicy):
```

Comment:
[Question] - and for example this policy is not dependent from any module. In conclusion, we need to analyze the policies needed for each module, those that are generic and how they are assigned

Reply:
If a service policy doesn't depend on the other module and should be included every single time then it should be part of the core.